base/sparse/cholmod.jl

```diff
@@ -869,12 +880,12 @@ function (::Type{Sparse}){Tv<:VTypes}(m::Integer, n::Integer,
         end
     end
 
-    o = allocate_sparse(m, n, length(nzval), iss, true, stype, Tv)
+    o = allocate_sparse(m, n, colptr[n + 1], iss, true, stype, Tv)
```

Comment:
Should the third argument to `allocate_sparse` be the number of stored entries or one more than the number of stored entries?